pkg/operator/resource/resourceapply/storage.go

```diff
@@ -173,6 +182,24 @@ func ApplyCSIDriver(ctx context.Context, client storageclientv1.CSIDriversGetter
 	return actual, true, err
 }
 
+func validateRequiredCSIDriverLabels(required *storagev1.CSIDriver) error {
+	supportsEphemeralVolumes := false
+	for _, mode := range required.Spec.VolumeLifecycleModes {
+		if mode == storagev1.VolumeLifecycleEphemeral {
+			supportsEphemeralVolumes = true
```

Comment:
once you found it you could break the loop to save a few cycles, no?

Reply:
yes, updated